AmplifyPlugins/API/Podfile.lock

```diff
@@ -52,7 +52,7 @@ DEPENDENCIES:
   - SwiftLint
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
+  trunk:
```

Comment:
not sure how to make this not change from the cdn to trunk, but `pod repo` shows 
```
trunk
- Type: CDN
- URL:  https://cdn.cocoapods.org/
- Path: /Users/<username>/.cocoapods/repos/trunk
```
so i think is fine